packages/valory/connections/abci/tests/test_fuzz/base.py

```diff
@@ -84,20 +90,23 @@ def setUpClass(cls) -> None:
         cls.agent_process = cls.run_agent()
 
         enforce(
-            cls.is_running(cls.agent_process, cls.AGENT_TIMEOUT),
-            f"The agent was not started in the defined timeout ({cls.AGENT_TIMEOUT}s)",
+            cls.is_running(cls.agent_process, cls.AGENT_TIMEOUT_SECONDS),
+            f"The agent was not started in the defined timeout ({cls.AGENT_TIMEOUT_SECONDS}s)",
         )
 
         enforce(cls.CHANNEL_TYPE is not None, "A channel type must be provided")
 
         cls.channel = cls.CHANNEL_TYPE(**cls.CHANNEL_ARGS)
         cls.mock_node = MockNode(cls.channel)
         cls.mock_node.connect()
+        logging.disable(logging.INFO)
```

Comment:
disable info logging for fuzzer tests because it's produces a lot of unnecessary output